cmd/neofs-cli/modules/object/head.go

```diff
@@ -140,6 +140,17 @@ func printContainerID(cmd *cobra.Command, recv func() (cid.ID, bool)) {
 
 func printHeader(cmd *cobra.Command, obj *object.Object) error {
 	printObjectID(cmd, obj.ID)
+	if signature := obj.Signature(); signature != nil {
+		cmd.Print("ID signature:\n")
+
+		// TODO(@cthulhu-rider): #1387 implement and use another approach to avoid conversion
```

Comment:
Let's use `@carpawell` here.

Reply:
well, in fact, i have no idea what is it about, i just moved that from one place to another

but ok, changed that